santa-tracker/mock-google-services.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Does this file help as is or does this need to be wired in somewhere?

Reply:
I wanted to avoid putting it under `google-services.json` in case someone wants to actually run the app using Google Play services. If we don't care about that, maybe I can just put it there. What do you think?

Reply:
How about we comment out the google services plugin altogether? The `google-services.json` is in `.gitignore`, and our performance test infrastructure creates the file when it does the checkout of the project.

Reply:
Even better. Let's do that. I'll upload a new commit to this PR, and also PR for `agp-3.6.0-java` branch. Sounds good?

Reply:
Sounds good. Can you remove the `google-services.json` from this PR?

Reply:
Done. Also, I've removed Google Play services plugin.